docker-compose.yml

```diff
@@ -58,8 +58,11 @@ services:
       - DJANGO_ES_URLS=http://elasticsearch:9200
       - DATABASE_URL=postgres://buildhub2:buildhub2@db/buildhub2
       - DEVELOPMENT=1
+      - GOOGLE_APPLICATION_CREDENTIALS=/tmp/credentials
     volumes:
       - $PWD:/app
+      # if credentials do not exist, mount an arbitrary file
+      - ${GOOGLE_APPLICATION_CREDENTIALS:-./README.rst}:/tmp/credentials
```

Comment:
What's the need for this? Can we just default `GOOGLE_APPLICATION_CREDENTIALS` to nothing instead?

Reply:
I've run into issues with CircleCI throwing an error if the path is too short (i.e. `./` or ` `), but this piece of code might be premature.